_posts/2021-11-18-real-time-and-historical-ad.markdown

```diff
@@ -0,0 +1,68 @@
+
+The user interface for viewing historical results is similar to the real-time results, but with a few extra helpful features:
+
+- Users always have the option to run a new historical analysis and select a new date range, by clicking on the “Modify historical analysis range” button, highlighted in red below. _Note that only the **latest** historical analysis job will be persisted and visible on this page. Previous runs and anomaly results will be discarded._
+
+  ![Modify historical range]({{ site.baseurl }}/assets/media/blog-images/2021-11-16-real-time-and-historical-ad/modify-historical-range.png){: .img-fluid }
+
+- As the historical analysis is running, the page will automatically refresh to show partial results, along with a progress bar. When the detection job is finished, users can zoom and view the anomaly results, as well as aggregate them on-the-fly by clicking on the “Daily max” / “Weekly max” / “Monthly max” buttons. Currently, only max anomaly grade aggregations are supported.
+
+  ![Raw historical results]({{ site.baseurl }}/assets/media/blog-images/2021-11-16-real-time-and-historical-ad/historical-results-raw.png){: .img-fluid }
+
+  ![Aggregated historical results]({{ site.baseurl }}/assets/media/blog-images/2021-11-16-real-time-and-historical-ad/historical-results-daily.png){: .img-fluid }
+
+There are a few other changes to the Anomaly Detection Dashboards plugin worth mentioning:
+
+- On the detector list page, an additional column related to historical analysis has been added. If there is any existing historical analysis found for a detector, a “View results” link will appear, which will navigate to the “Historical analysis” tab on the detector’s detail page.
+
+  ![Detector list page]({{ site.baseurl }}/assets/media/blog-images/2021-11-16-real-time-and-historical-ad/detector-list.png){: .img-fluid }
+
+- The sample detectors page has been moved into a new “Overview” page, which provides additional high-level information about anomaly detection. This page can always be accessed by clicking on the “Anomaly detection” title in the side navigation bar.
+
+  ![Overview page]({{ site.baseurl }}/assets/media/blog-images/2021-11-16-real-time-and-historical-ad/overview.png){: .img-fluid }
+
+### Conclusion
+
+In this blog post, we discussed two new features released in OpenSearch 1.1 - anomaly detection for historical data and universal flow in OpenSearch Dashboards that enables users to just configure once and detect anomalies on real-time and historical data. Anomaly detection for historical data helps users analyze and identify trends and patterns in past data. Users can use these insights to take better decisions, improve planning, and boost the overall operational efficiency of their applications.
+
+We are excited for the future of anomaly detection in OpenSearch and welcome you to come join in and contribute with us in building many more exciting features in anomaly detection and machine learning.
```

Comment:
'We are excited for the future of anomaly detection in OpenSearch' doesn't add much and "we are exited" is a bit overused in Amazonian writing. I think you can safely drop it.